content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -151,6 +151,7 @@ different Kubernetes components.
 | `ServerSideApply` | `false` | Alpha | 1.14 | |
 | `ServiceLoadBalancerFinalizer` | `false` | Alpha | 1.15 | |
 | `ServiceNodeExclusion` | `false` | Alpha | 1.8 | |
+| `StartupProbe` | `false` | Alpha | 1.16 | |
```

Comment:
@sftim I have also updated the name of the feature gate to match https://github.com/kubernetes/kubernetes/blob/master/pkg/features/kube_features.go#L498